### ISSUE TITLE
tf.image.resize_images cannot be applied to NCHW format directly?

### ISSUE THREAD

**Issue by S-aiueo32** [tensorflow/tensorflow#22016]
Referencing TensorFlow's Performance Guide, It seems NCHW data format is better than NHMC data format in many situations.
Some existing models has already been implemented in NCHW data format, especially using keras, but tf.image.resize_images cannot be applied to NCHW format directly, I think.
Are there any way excepting that we implement by ourselves?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by wt-huang** [comment]
@S-aiueo32  You can use ` tf.transpose`

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by melgor** [comment]
I have also encounter this problem. `tf.transpose` works but it make the model much more slower. Using trace of MTCNN network, look like 'tf.transpose' take 40% of time. So for me this is still probemalic to use resize operation with NHWC format
